sam_view.c

```diff
@@ -261,14 +262,14 @@ static inline int check_sam_write1(samFile *fp, const sam_hdr_t *h, const bam1_t
 
 int main_samview(int argc, char *argv[])
 {
-    int c, is_header = 0, is_header_only = 0, ret = 0, compress_level = -1, is_count = 0, has_index_file = 0;
+    int c, is_header = 0, is_header_only = 0, ret = 0, compress_level = -1, is_count = 0, has_index_file = 0, no_pg = 0;
```

Comment:
The code for all these --no-PG options would be clearer if the variable was `add_pg` starting at 1 and reset to 0 by the option.